app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UserSessionsController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
+      redirect_back_or_to(:users, notice: 'Login successful')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.